app/models/pusher.rb

```diff
@@ -142,8 +142,8 @@ def set_info_checksum
 
   def republish_notification(version)
     if version.indexed?
-      notify("Repushing of gem versions is not allowed.\n" \
-             "Please use `gem yank` to remove bad gem releases.", 409)
+      notify("Repushing of gem versions is not allowed.\n"
+             "Please bump the version number and push a new different release.", 409)
```

Comment:
I think this message can be unified. As a user unfamiliar with RubyGems looking to undo a bad push, the error saying to use `gem yank` is a helpful step. 

Maybe something like:

    Repushing of gem versions is not allowed. Please use `gem yank` to remove bad gem releases and push a newer version.

Reply:
I'd be in favor of first telling one to not yank but push new (as per experience yanking is really often unpopular to the community). However, indicating the existance of such a possibility is a good idea IMHO. Hence I have this counter offer @colby-swandale :
```suggestion
             "Please bump the version number and push a new different release.\n"
             "See `gem yank` if you want to get rid of the bad release.", 409)
```

Reply:
That sounds like a good middle ground to me!

Reply:
@colby-swandale what do you say? :)

Reply:
This sounds good. My only bit of feedback would be to replace `rid` with `unpublish`.

Reply:
Done :)

Reply:
Cheers!